src/jdk.jlink/share/classes/jdk/tools/jlink/internal/plugins/DocumentedPlugin.java

```diff
@@ -0,0 +1,26 @@
+package jdk.tools.jlink.internal.plugins;
+
+import jdk.tools.jlink.plugin.Plugin;
+
+public abstract class DocumentedPlugin implements Plugin {
```

Comment:
I agree with Alan that `AbstractPlugin` is better.

This should also provide the default implementation of `getArgumentsDescription()` such that the plugins don't need to implement.